src/Avalonia.Themes.Fluent/Controls/SplitButton.xaml

```diff
@@ -0,0 +1,373 @@
+﻿<Styles xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        x:CompileBindings="True">
+  <Design.PreviewWith>
+  </Design.PreviewWith>
+
+  <Styles.Resources>
+    <x:Double x:Key="SplitButtonPrimaryButtonSize">32</x:Double>
+    <x:Double x:Key="SplitButtonSecondaryButtonSize">32</x:Double>
+  </Styles.Resources>
+
+  <Style Selector="SplitButton">
+    <Setter Property="Background" Value="{DynamicResource SplitButtonBackground}" />
+    <Setter Property="Foreground" Value="{DynamicResource SplitButtonForeground}" />
+    <Setter Property="BorderBrush" Value="{DynamicResource SplitButtonBorderBrush}" />
+    <Setter Property="BorderThickness" Value="{DynamicResource SplitButtonBorderThemeThickness}" />
+    <Setter Property="HorizontalAlignment" Value="Left" />
+    <Setter Property="VerticalAlignment" Value="Center" />
+    <!--<Setter Property="FontFamily" Value="{ThemeResource ContentControlThemeFontFamily}" />
+    <Setter Property="FontWeight" Value="Normal" />
+    <Setter Property="FontSize" Value="{ThemeResource ControlContentThemeFontSize}" />
```

Comment:
FontFamily, FontSize and FontWeight setters were removed in most places. These exist in the WinUI default style but I consider it better to let these be inherited. Is this correct?

Reply:
Yes

Reply:
I think a lot of Fluent styles need to be fixed and remove these then. That is probably the cause of the bug reports I see about changing text size for the entire app window.